src/api/modules/utils.ts

```diff
@@ -282,118 +182,6 @@ export const yarnPath = `"${require.resolve('yarn/bin/yarn')}"`
 export const formatNano = (nanoseconds: number): string =>
   `${(nanoseconds / 1e9).toFixed(0)}s ${((nanoseconds / 1e6) % 1e3).toFixed(0)}ms`
 
-export const runYarn = (relativePath: string, force: boolean) => {
-  log.info(`Running yarn in ${chalk.green(relativePath)}`)
-  const root = getAppRoot()
-  const command = force
-    ? `${yarnPath} --force --non-interactive --ignore-engines`
-    : `${yarnPath} --non-interactive --ignore-engines`
-  execSync(command, { stdio: 'inherit', cwd: resolvePath(root, relativePath) })
-  log.info('Finished running yarn')
-}
-
-export const runYarnIfPathExists = (relativePath: string) => {
-  const root = getAppRoot()
-  const pathName = resolvePath(root, relativePath)
-  if (existsSync(pathName)) {
-    try {
-      runYarn(relativePath, false)
-    } catch (e) {
-      log.error(`Failed to run yarn in ${chalk.green(relativePath)}`)
-      throw e
-    }
-  }
-}
-
-const formatAppId = (appId: string) => {
-  const [appVendor, appName] = R.split('.', appId)
-  if (!appName) {
-    // Then the app is an 'infra' app.
-    const [infraAppVendor, infraAppName] = R.split(':', appId)
-    if (!infraAppName) {
-      return appId
-    }
-    return `${chalk.blue(infraAppVendor)}:${infraAppName}`
-  }
-  return `${chalk.blue(appVendor)}.${appName}`
-}
-
-const cleanVersion = (appId: string) => {
-  return R.compose<string, string[], string, string>(
-    (version: string) => {
-      const [pureVersion, build] = R.split('+build', version)
-      return build ? `${pureVersion}(linked)` : pureVersion
-    },
-    R.last,
-    R.split('@')
-  )(appId)
-}
-
-export const matchedDepsDiffTable = (title1: string, title2: string, deps1: string[], deps2: string[]) => {
```

Comment:
Hey @arturpimentel can you keep this function on the code. This plugin is actually using it https://github.com/vtex/cli-plugin-deps/blob/master/src/modules/deps/diff.ts#L1 😢

Reply:
Oops! I'll restore this 😬

Reply:
Thanks, bro, sorry about it. Toolbelt has a bunch of projects that depend on it implicitly. Super tricky to spot this, since it spans multiple repos. I guess the solution would be to have regression testing on CI. But this is still to be implemented.

Reply:
I had to rebase this branch onto `3.x` and then push force, so if you check the commit history it will just seem as I have never removed it haha. Anyway, in https://github.com/vtex/toolbelt/pull/1092/commits/87e54aa74f10e867edf2705d63dbd20d1e8f4a08 you'll find that I just removed the duplicate utils from `modules/utils.ts`.